app/controllers/posts_controller.rb

```diff
@@ -17,7 +17,14 @@ def create
     @post = Post.new(post_params)
     @post.user_id = current_user.id
     if @post.save
-      # here is a potential place to add Twilio API call to send text messages 
+      # Here is a potential place to add Twilio API call to send text messages 
+
+      # Construct the API URL here, taking care to use Rails.application.secrets.bitly_api_key and
+      # Rails.application.secrets.bitly_username, for the login, and Rails route helpers the post's url
+      bitly_url = "http://api.bitly.com/..."
+
+      # You can use a simple get call via Net::HTTP like this, once the URL is constructed
+      resp = JSON.parse(Net::HTTP.get URI(bitly_url))
```

Comment:
Interesting... Bitly doesn't offer a Ruby client like Twilio? The client is way cleaner :)